My_work_2/Фигуры Лиссажу.py

```diff
@@ -14,7 +14,9 @@
 fix = 0
 fiy = 0
 
-t = np.arange(-3.14, 3.15, 0.01)
+step = 0.001
+
+t = np.arange(-np.pi, np.pi+step, step)
```

Comment:
np.linspace is a more common way to get homogeneous distribution over the real values range.